driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/InstalledIntelliJ.scala

```diff
@@ -53,7 +47,9 @@ final class InstalledIntelliJ(val root: Path, config: DriverConfig) {
       val connection = JsonRpcConnection.from(socket)
 
       val driver = ProbeDriver.start(connection)
-
+      val stringFromConfig = probeConfig.source.value
```

Comment:
it would be nice to put all the newly added logic into a method

Reply:
how about higher order function then